src/uk/gov/hmcts/pipeline/EnvironmentApprovals.groovy

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.pipeline
+
+class EnvironmentApprovals {
+  def steps
+  static final String GITHUB_CREDENTIAL = 'jenkins-github-hmcts-api-token'
+  static def approvalsMap
```

Comment:
shouldn't need to include the type in a variable name?
if you're tempted to do that and IDE can't do it for you maybe put the actual type in groovy?
I assume it's

```suggestion
  private static Map<String, Object> approvalsMap
```

Reply:
cut'n'paste from TeamNames 😄 - probably better called environmentApprovals anyway